measured.go

```diff
@@ -416,24 +425,20 @@ func (m *Measured) submitError(connID string, err error, phase string) {
 		Error: e,
 		Phase: phase,
 	}
-	log.Tracef("Submiting error %+v", er)
-	select {
-	case m.chStat <- er:
-	default:
-		log.Error("Failed to submit error, channel busy")
-	}
+	log.Tracef("Submitting error %+v", er)
+	m.chStat <- er
 }
 
 func (m *Measured) submitTraffic(connID string, BytesIn uint64, BytesOut uint64) {
+	if atomic.LoadInt32(&m.stopped) == 1 {
+		log.Trace("Measured stopped, not submitting traffic")
+		return
+	}
 	t := &Traffic{
 		ID:       connID,
 		BytesIn:  BytesIn,
 		BytesOut: BytesOut,
 	}
-	log.Tracef("Submiting traffic %+v", t)
-	select {
-	case m.chStat <- t:
-	default:
-		log.Error("Failed to submit traffic, channel busy")
```

Comment:
I don't quite understand. With the checking of `stopped` flag, `log.Error` is removed.